unit-tests/native/src/test/scala/scala/scalanative/regex/ParserTest.scala

```diff
@@ -168,7 +168,7 @@ class ParserTest {
       "cc{0x0-0x2f 0x3a-0x40 0x5b-0x5e 0x60 0x7b-0x17e 0x180-0x2129 0x212b-0x10ffff}"
     ),
     Array("[^\\\\]", "cc{0x0-0x5b 0x5d-0x10ffff}"),
-    //  { "\\C", "byte{}" },  // probably never
+    //	{ "\\C", "byte{}" },  // probably never
```

Comment:
Probably spaces were replaced here with tabs, can you please revert this?

Reply:
Done